tests/.pest/snapshots/Countries/PolandTest/it_can_calculate_polish_holidays.snap

```diff
@@ -0,0 +1,54 @@
+[
+    {
+        "name": "Nowy Rok",
+        "date": "2024-01-01"
+    },
+    {
+        "name": "Święto Trzech Króli",
+        "date": "2024-01-06"
+    },
+    {
+        "name": "Święto Pracy",
+        "date": "2024-05-01"
+    },
+    {
+        "name": "Święto Konstytucji 3 Maja",
+        "date": "2024-05-03"
+    },
+    {
+        "name": "Święto Wojska Polskiego, Wniebowzięcie Najświętszej Maryi Panny",
+        "date": "2024-08-15"
+    },
+    {
+        "name": "Wszystkich Świętych",
+        "date": "2024-11-01"
+    },
+    {
+        "name": "Święto Niepodległości",
+        "date": "2024-11-11"
+    },
+    {
+        "name": "Boże Narodzenie",
+        "date": "2024-12-25"
+    },
+    {
+        "name": "Boże Narodzenie",
+        "date": "2024-12-26"
+    },
+    {
+        "name": "Wielkanoc",
+        "date": "2024-03-31"
```

Comment:
All holidays should be in chronological order, so this one should be placed before "2024-05-01"